src/AppBundle/Service/RestHandler/Report/DeputyCostsEstimateReportUpdateHandler.php

```diff
@@ -29,11 +29,30 @@ public function handle(Report $report, array $data)
         $this
             ->updateHowCharged($report, $data)
             ->updateBreakdownEstimates($report, $data)
-            ->updateMoreInfo($report, $data);
+            ->updateMoreInfo($report, $data)
+            ->updateManagementCost($report, $data)
+        ;
 
         $report->updateSectionsStatusCache([Report::SECTION_PROF_DEPUTY_COSTS_ESTIMATE]);
     }
 
+    /**
+     * @param Report $report
+     * @param array $data
+     * @throws \Doctrine\ORM\OptimisticLockException
+     */
+    private function updateManagementCost(Report $report, array $data) {
+        if (array_key_exists('prof_deputy_management_cost_amount', $data)) {
+            $report->setProfDeputyCostsEstimateManagementCostAmount($data['prof_deputy_management_cost_amount']);
+
+            $report->updateSectionsStatusCache([
```

Comment:
I don't _think_ this call is necessary as it is called at the end of all updates on line 36

Reply:
Ah yeah good spot - I'll drop 👍